tests/Unit/Console/Worker/BytesTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Tests\Unit\Console\Worker;
+
+use Generator;
+use Patchlevel\EventSourcing\Console\Worker\Bytes;
+use Patchlevel\EventSourcing\Console\Worker\InvalidFormat;
+use PHPUnit\Framework\TestCase;
+
+final class BytesTest extends TestCase
+{
+    public function testParseError(): void
+    {
+        $this->expectException(InvalidFormat::class);
+
+        Bytes::parseFromString('505Foo');
+    }
+
+    /**
+     * @dataProvider validParseDataProvider
+     */
+    public function testValidParse(string $string, int $expectedBytes): void
+    {
+        $bytes = Bytes::parseFromString($string);
+
+        self::assertSame($expectedBytes, $bytes->value());
+    }
+
+    public function validParseDataProvider(): Generator
```

Comment:
Add the type here so that we dont need to add the error to the baseline?

Reply:
wenn you know how, go for it. try it :D

Reply:
Cant reproduce it in psalm.dev but this should be enough i think:
```php
    /**
     * @return Generator<array{0:string, 1:int}>
     */
```